### PR TITLE
bug fix for uploading image in SSF template

### PR DESCRIPTION
**Pull request by samsruti**
Link to the bug : http://sunflower.sahanafoundation.org/eden/project/task/344
GCI task link : http://www.google-melange.com/gci/task/view/google/gci2014/5829498209566720

Added a "Images detail" tab for better accessibility by the user. Now uploading image is working perfectly.

### CODE REVIEW

**Comment by hitesh96db** [comment]
I would suggest using the ImageCropWidget(which doesn't work with InlineComponents so you could keep that as a component tab)

File: private/templates/SSF/config.py [hunk 1]
```diff
@@ -825,6 +825,7 @@ def customise_pr_person_controller(**attr):
     has_permission = current.auth.s3_has_permission
     if has_permission("read", "pr_contact"):
         tabs.append((T("Contact Details"), "contacts"))
+        tabs.append((T("Image Details"), "image"))
```
Comment (hitesh96db) [inline private/templates/SSF/config.py]: "Images" would be better

File: private/templates/SSF/config.py [hunk 2]
```diff
@@ -862,14 +870,16 @@ def custom_prep(r):
                 msg_record_deleted = T("Contributor deleted"),
                 msg_list_empty = T("No Contributors currently registered")
             )
+                   
 
             # Custom Form (Read/Create/Update)
             from s3.s3forms import S3SQLCustomForm, S3SQLInlineComponent
-
-            crud_form = S3SQLCustomForm(
+            s3db.pr_image.image.widget = None
```
Comment (hitesh96db) [inline private/templates/SSF/config.py]: Use ImageCropWidget(will have to remove InlineComponent)

**Comment by ptressel** [comment]
Thanks!

**Pull request merged**